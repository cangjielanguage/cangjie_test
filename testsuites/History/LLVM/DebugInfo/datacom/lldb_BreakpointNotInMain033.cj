// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_BreakpointNotInMain033_expected.txt ../basecase/041_debuginfun_parentheses.cj
// EXEC: cjc -g 041_debuginfun_parentheses.cj
// EXEC: python3 DatacomTest.py lldb_BreakpointNotInMain033_expected.txt



