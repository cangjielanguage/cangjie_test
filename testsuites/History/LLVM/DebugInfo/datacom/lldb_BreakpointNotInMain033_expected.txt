lldb,lldb 041_debuginfun_parentheses,041_debuginfun_parentheses
b,b 8,041_debuginfun_parentheses.cj:8
b,b 9,041_debuginfun_parentheses.cj:9
r,r,-> 8
n,n,-> 9
cj_plocals,cj_plocals,kuonum = 5